imports/client/components/HuntListPage.jsx

```diff
@@ -210,6 +218,45 @@ class HuntModalForm extends React.Component {
           </div>
         </FormGroup>
 
+        <FormGroup>
+          <ControlLabel htmlFor={`${idPrefix}submit-template`} className="col-xs-3">
+            Submit URL template
+          </ControlLabel>
+          <div className="col-xs-9">
+            <FormControl
+              id={`${idPrefix}submit-template`}
+              type="text"
+              value={this.state.submitTemplate}
+              onChange={this.onSubmitTemplateChanged}
+              disabled={disableForm}
+            />
+            <HelpBlock>
+              If provided, this
+              {' '}
+              <a href="https://mustache.github.io/mustache.5.html">Mustache template</a>
+              {' '}
+              is used to generate the link to the guess submission page. It gets as context a
+              {' '}
+              <a href="https://developer.mozilla.org/en-US/docs/Web/API/URL">parsed URL</a>
+              {', '}
+              providing variables like
+              {' '}
+              <code>hostname</code>
+              {' '}
+              or
+              {' '}
+              <code>pathname</code>
+              {'. '}
+              As an example, setting this to
+              {' '}
+              <code>{'{{origin}}/submit{{pathname}}'}</code>
```

Comment:
based on my other comment about not disabling autoescaping, make this template have triple-`{{{` for the fields